packages/astro/src/integrations/index.ts

```diff
@@ -221,17 +221,21 @@ export async function runHookConfigDone({
 								logger
 							);
 							for (const [featureName, supported] of Object.entries(validationResult)) {
-								if (!supported) {
+								// If `supported` / `validationResult[featureName]` only allows boolean,
+								// in theory 'asstes' false, doesn't mean that the feature is not supported, but rather that the chosen image service is unsupported
+								// in this case we should not show an error, that the featrue is not supported
+								// if we would refactor the validation to support more than boolean, we could still be able to differentiate between the two cases
+								if (!supported && featureName !== 'assets') {
 									logger.error(
 										'astro',
 										`The adapter ${adapter.name} doesn't support the feature ${featureName}. Your project won't be built. You should not use it.`
 									);
 								}
 							}
 							if (!validationResult.assets) {
-								logger.info(
+								logger.warn(
 									'astro',
-									`The selected adapter ${adapter.name} does not support Sharp or Squoosh for image processing. To ensure your project is still able to build, image processing has been disabled.`
+									`The selected adapter ${adapter.name} does not support image processing. To ensure your project continues to build successfully, the image service has been automatically switched to the 'noop' option, effectively disabling image processing.`
```

Comment:
We don't need to mention `sharp` or `squoosh` again, as we've already stated in the log line above that the current adapter is incompatible with them.

Reply:
Just checking that people feel that "does not support image processing/disabling image processing" is clear enough that it won't make people think they somehow can't even use/have images in their project. (Out of context of the inner workings of the image services API, "processing" could literally mean "handling/being able to accept".)

It might be worth something slightly more reassuring that mentions what WILL happen to/with your images (which sounds less like images are somehow a problem): they will not be optimized, they will be copied into your build as is, this allows you to use the image component and `getImage()` function although the functionality will be limited to preventing CLS and will not optimize or otherwise process your images.

Reply:
@sarah11918 do you have a suggestion for me, or do we wait for other opinions?

Reply:
We can say something like "The selected adapter ${adapter.name} doesn't support image optimization, although they will still be served correctly. To ensure your project ..."